marketplace/gatsby-preview/extension.json

```diff
@@ -7,19 +7,20 @@
       {
         "id": "previewUrl",
         "type": "Symbol",
-        "name": "The URL of your Gatsby Cloud instance",
+        "name": "The url of your Gatsby Cloud site",
```

Comment:
I'd keep the "URL" capitalized: https://en.wikipedia.org/wiki/URL

Reply:
Done.